gcloud-java-examples/README.md

```diff
@@ -87,6 +87,20 @@ To run examples from your command line:
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.StorageExample" -Dexec.args="download <bucket_name> test.txt"
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.StorageExample" -Dexec.args="delete <bucket_name> test.txt"
     ```
+ * Here's an example run of `Stat`.
+
+     Before running the example, go to the [Google Developers Console][developers-console] to ensure that Google Cloud Storage API is enabled and that you have a bucket with a file in it.
+     Compile the JAR with:
+    ```
+    mvn package -DskipTests
+    ```
+    Then run the sample with:
+    ```
+    java -cp gcloud-java-contrib/gcloud-java-nio/target/gcloud-java-nio-0.1.4-SNAPSHOT-jar-with-dependencies.jar:gcloud-java-examples/target/gcloud-java-examples-0.1.4-SNAPSHOT.jar com.google.gcloud.examples.Stat --check
```

Comment:
`gcloud-java-nio-0.1.4-SNAPSHOT-jar-with-dependencies.jar` that is the assembly plugin's artifcat and not the shaded one, right?